livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/tts.py

```diff
@@ -47,20 +47,22 @@ def __init__(
         base_url: str | None = None,
         api_key: str | None = None,
         client: openai.AsyncClient | None = None,
+        timeout: float = 0,
     ) -> None:
         super().__init__(
             capabilities=tts.TTSCapabilities(
                 streaming=False,
             ),
             sample_rate=OPENAI_TTS_SAMPLE_RATE,
             num_channels=OPENAI_TTS_CHANNELS,
+            timeout=timeout,
```

Comment:
Seems like timeout=0 isn't a good default when not using the FallbackAdapter?

Reply:
we kinda have to keep it this way, timeout=0 just disables the timeout which is the way it has been. for backwards compatibility we have to default to 0